src/components/AppRouter/index.tsx

```diff
@@ -7,7 +7,8 @@ import { Navigation } from "../Navigation"
 
 export const HOST = process.env.REACT_APP_HOST
 export const PORT = process.env.REACT_APP_APIPORT
-
+export const CORS_ANYWHERE = process.env.REACT_APP_CORS_ANYWHERE_PORT
```

Comment:
@Alexandria I see two variables called `CORS_ANYWHERE`, could one of them import from the other?